ietf/templates/base/icons.html

```diff
@@ -12,8 +12,8 @@
 <link rel="manifest" href="/site.webmanifest">
 <link rel="mask-icon"
       href="{% static 'ietf/images/ietf-logo-nor-mask.svg' %}"
-      {% if server_mode and server_mode == 'production' %}color="#ffffff"{% else %}color="#ffc4c4"{% endif %}>
+      content="{% if server_mode and server_mode == 'production' %}#ffffff{% else %}#ffc4c4{% endif %}">
```

Comment:
I think the `content` should be `color` for the `link rel="mask-icon"` ([Apple developer docs for pinned tabs](https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/pinnedTabs/pinnedTabs.html)).

Reply:
Good catch, thanks